pkg/collector/node_energy.go

```diff
@@ -80,16 +135,49 @@ func (v *NodeEnergy) ResetCurr() {
 	v.SensorEnergy.ResetCurr()
 }
 
-func (v *NodeEnergy) SetValues(sensorEnergy map[string]float64, pkgEnergy map[int]source.RAPLEnergy, totalGPUDelta uint64, usage map[string]float64) {
+func (v NodeEnergy) sumUsage(podMetricValues [][]float64) (nodeUsageValues []float64, nodeUsageMap map[string]float64) {
+	nodeUsageValues = make([]float64, len(metricNames))
+	nodeUsageMap = make(map[string]float64)
+	podNumber := len(podMetricValues)
+	for metricIndex, metric := range metricNames {
+		nodeUsageMap[metric] = 0
+		for podIndex := 0; podIndex < podNumber; podIndex++ {
+			nodeUsageMap[metric] += podMetricValues[podIndex][metricIndex]
+			nodeUsageValues[metricIndex] += podMetricValues[podIndex][metricIndex]
+		}
+	}
+	return
+}
+
+func (v *NodeEnergy) SetValues(sensorEnergy map[string]float64, pkgEnergy map[int]source.RAPLEnergy, totalGPUDelta uint64, podMetricValues [][]float64) {
+	nodeUsageValues, nodeUsageMap := v.sumUsage(podMetricValues)
+	v.Usage = nodeUsageMap
 	for sensorID, energy := range sensorEnergy {
-		v.SensorEnergy.AddStat(sensorID, uint64(energy))
+		v.SensorEnergy.AddAggrStat(sensorID, uint64(energy))
```

Comment:
@sunya-ch 
The measured data added as AggrStat and the estimated as the CurrStat
So, we only estimate the current but measure the aggregation?
Is that consistent?

Reply:
Value from sensor is aggregated value. Value from estimation is current value (estimated from current value).

Reply:
Ok, thanks!
Maybe we should consider collect the CurrStat in both cases for consistency...